java/src/jmri/jmrit/operations/locations/JmritOperationsLocationsBundle_ja_JP.properties

```diff
@@ -5,124 +5,124 @@
 # Japanese properties for the jmri.jmrit.operations.locations
 # Translated by Sakae Akanuma <sakaeakanuma@gmail.com>
 
-TitleLocationsTable = \u30ED\u30B1\u30FC\u30B7\u30E7\u30F3
-TitleLocationEdit   = \u30ED\u30B1\u30FC\u30B7\u30E7\u30F3\u7DE8\u96C6
-TitleLocationAdd    = \u30ED\u30B1\u30FC\u30B7\u30E7\u30F3\u8FFD\u52A0
-TitleScheduleEdit   = \u4FE1\u53F7\u6A5F {0}\u306E\u30B9\u30B1\u30B8\u30E5\u30FC\u30EB\u7DE8\u96C6
-TitleScheduleAdd    = \u30B5\u30A4\u30C7\u30A3\u30F3\u30B0 {0}\u306E\u30B9\u30B1\u30B8\u30E5\u30FC\u30EB\u3092\u8FFD\u52A0
-TitleSchedulesTable = \u30B9\u30B1\u30B8\u30E5\u30FC\u30EB
+TitleLocationsTable = \u30ed\u30b1\u30fc\u30b7\u30e7\u30f3
```

Comment:
I was unaware all these translations were stored as upper case escapes. Following agreed styling, but this will hide the original edits. The real change to each is limited to commenting out the Length key (moved to jmrit.Bundle to be shared with Panels end Tables).